specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/WafListAllPolicies.json

```diff
@@ -87,6 +87,51 @@
                     }
                   ],
                   "action": "Block"
+                },
+                {
+                  "name": "RateLimitRule3",
+                  "priority": 3,
+                  "rateLimitDuration": "OneMin",
+                  "rateLimitThreshold": 10,
+                  "ruleType": "RateLimitRule",
+                  "matchConditions": [
+                    {
+                      "matchVariables": [
+                        {
+                          "variableName": "RemoteAddr",
+                          "selector": null
```

Comment:
Will this come back if it is null?

Reply:
I assume you meant come back from NRP k/v store. If so, the answer is no.

Reply:
Just realized that the "selector" your were pointing to is under "matchConditions". This section I copied over as is from the previous custom rules in the examples. They all have "selector": null in them. Do you want me to remove this across all examples? I have updated the examples such that the rate limit custom rule have match conditions that make sense within the context of each example.